ui/src/environments/environment.local.ts

```diff
@@ -11,5 +12,8 @@ export const environment = {
   publicUiUrl: 'http://localhost:4201',
   debug: true,
   gaId: 'UA-112406425-5',
-  enableTemporal: true
+  zendeskHelpCenterUrl: 'http://aousupporthelp.zendesk.com/hc/',
```

Comment:
nit: Is the trailing / needed? If we wanted to construct other URLs from this later it would be nice if it weren't here.

Reply:
Done.